spec/lib/data_update_scripts/append_collective_noun_to_community_name_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20201228194641_append_collective_noun_to_community_name.rb",
+)
+
+describe DataUpdateScripts::AppendCollectiveNounToCommunityName do
+  context "when collective_noun_disabled is false" do
+    it "appends the collective_noun to the community_name" do
+      SiteConfig.collective_noun = "Club"
+      described_class.new.run
+      expect(SiteConfig.collective_noun_disabled).to eq(false)
+      expect(SiteConfig.community_name).to eq("DEV(local) Club")
```

Comment:
I could also change this to mirror the code snippet below if folks think it's more readable and/or prefer it written this way:
```
      expect(SiteConfig.collective_noun_disabled).to eq(false)
      expect do
        described_class.new.run
      end.to change(SiteConfig, :community_name).from("DEV(local)").to("DEV(local) Club")
```
(The same goes for the test below!)

Reply:
I like this a little better, and I think I've mostly seen this as a convention in our codebase!